docs/diagrams/CommandInheritance.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+left to right direction
+abstract Command
+Command <|-- AddCommand
+Command <|-- AddMemberCommand
+Command <|-- AddTransactionCommand
```

Comment:
You can try to use the `hide circle` command to hide the 'circle C', as our textbook does not mention that we need to have those icons